### PR TITLE
Added URL and images in former-board

### PR DESCRIPTION
**Pull request by tharun143**
Signed-off-by: tharun143 <tharunravuri2000@gmail.com>

### CODE REVIEW

**Comment by tharun143** [comment]
@klumb Added images with full URL in the file for issue #315

**Comment by tharun143** [comment]
@klumb Are there any suggestions?

**Comment by GeorgLink** [comment]
Hi @tharun143, I appreciate your enthusiasm and drive to participate. Please allow more time for replies from other community members. If you are looking for a rule of thumb, I'd say to wait three days before bumping an issue or pull request. I'm saying this, because CHAOSS is a global community that spans almost all timezeones and we want to give everyone a chance to respond.

File: About/former-board.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+
+# Former Board Members:
+
+
+
+## Jono Bacon
+
+![Jono Bacon](website/About/images/jono.jpg)
```
Comment (GeorgLink) [inline About/former-board.md:9]: For the images, we need absolute paths. Take a look at the CHAOSScon website for an example.

> https://github.com/chaoss/website/blob/master/CHAOSScon/2020EU/speakers.md

**Comment by tharun143** [comment]
@GeorgLink I will change it and sorry for the trouble. I am excited a little bit for contributing towards CHAOSS community.

**Pull request closed without merging**